### PR TITLE
[BOX] Changes medbay doors unrestricted access

### PR DESCRIPTION
**Pull request by AMyriad**
# Document the changes in your pull request
This changes the silly arrows in the main medbay area and alters what doors have unrestricted access from the inside to allow patients to exit from both left and right entrances, rather than just the right one in a sort of "come in on the left, exit on the right" style.

# Why is this good for the game?
Helps those patients who just finished getting treated trying to exit on the left side, which since the sleeper/stasis bed are there, a **lot** of people try to do that. I get the system of a designated entrance and designated exit, but it doesn't always work like that.

People will come in from, and exit to both sides, and I've seen on too many occasions people annoyingly can't exit from one side, but can from another. I know it's not hard to follow basic directions, but this change is for convenience sake, as those directions are arbitrary, and aren't going to be respected or followed in real situations.

# Testing
image

# Changelog
:cl:  
mapping: Patients can now exit from both left and right doors on Box medbay
/:cl:

### CODE REVIEW

**Comment by MajManatee** [comment]
ya dont drive on both sides of the road

**Comment by ynot01** [comment]
is there no order anymore

**Comment by JamieD1** [comment]
Considering I was a huge part of wanting this, cant people just not be dumb?

**Comment by patpol4** [comment]
hugbox

**Comment by AMyriad** [comment]
most of my reasoning is said in the "Why is this good for the game?" section so I won't repeat myself
do, however, bear in mind that most stations already do this

gax has this, despite what the arrow decals show:
image

icemeta does this:
image

and so does donut:
image

box actually stands out here, and it's a bit silly that they do :p

**Comment by patpol4** [comment]
None of these have two entrances

**Comment by neeshacark** [comment]
> None of these have two entrances

gax literally has two entrances

**Review by neeshacark — APPROVED** [review]
